frontend/src/static/js/components/-NEW-/ItemList.js

```
@@ -8,95 +8,107 @@ import PageStore from '../../pages/_PageStore.js';
+	hideViews: false,
+	hideAuthor: false,
+	hidePlaylistOptions: true,
+	hidePlaylistOrderNumber: true,
+	hideAllMeta: false,
+	inPlaylistView: false,
+	inPlaylistPage: false,
+	playlistActiveItem: 1,
+	playlistId: void 0,
+	/* ################################################## */
+	maxItems: 99999,
+	pageItems: 48,
```

Biome (2.4.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)